### PR TITLE
Mark SPECs as draft until they have two endorsements

### PR DESCRIPTION
**Pull request by jarrodmillman**

### CODE REVIEW

File: layouts/partials/post_meta/spec_meta.html [hunk 1]
```diff
@@ -41,7 +41,22 @@
             </td>
           {{- end -}}
         </tr>
-      {{- end -}}
+      {{- $N := len (or $val "") }} 
+      {{- if (and (eq $attr "endorsed-by") (le $N 1)) }}
```
Comment (stefanv) [inline layouts/partials/post_meta/spec_meta.html:45]:
```suggestion
      {{- if (and (eq $attr "endorsed-by") (lt (len $val) 2)) }}
```

File: layouts/partials/post_meta/spec_meta.html [hunk 2]
```diff
@@ -41,7 +41,22 @@
...
+          <tr class="field">
```
Comment (stefanv) [inline layouts/partials/post_meta/spec_meta.html:46]:
```suggestion
          {{ /* add blank line to table */ }}
          <tr class="field">
```

File: layouts/partials/post_meta/spec_meta.html [hunk 3]
```diff
@@ -41,7 +41,22 @@
...
+              <td class="field-name">&nbsp;</td>
+              <td class="field-value"></td>
+          </tr>
+          <tr class="field" style="background-color:crimson;color:white;">
```
Comment (stefanv) [inline layouts/partials/post_meta/spec_meta.html:50]:
```suggestion
          {{ /* add warning that this is a draft, i.e. has not been endorsed by 2 projects or more */ }}
          <tr class="field" style="background-color:crimson;color:white;">
```

File: layouts/partials/post_meta/spec_meta.html (hunk 1 above)
Comment (jarrodmillman) [reply layouts/partials/post_meta/spec_meta.html:45]: That didn't work, but this did
```
{{- if (and (eq $attr "endorsed-by") (lt (len (or $val "")) 2)) }}
```

File: layouts/partials/post_meta/spec_meta.html (hunk 2 above)
Comment (jarrodmillman) [reply layouts/partials/post_meta/spec_meta.html:46]: Hugo complained about the spaces (i.e., `{{ /*` and `*/ }}`).  But it worked without the spaces (i.e., `{{/*` and `*/}}`).

**Pull request merged**